README.md

```diff
@@ -0,0 +1 @@
+# Fonto Editor AI refernece configuration
```

Comment:
Shouldn't we here describe how to integrate the add-on. For example that the route needs to be "registered"

Reply:
Yup, I still had to do that.